src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationOptions.cs

```diff
@@ -58,7 +58,7 @@ public OpenIdConnectAuthenticationOptions(string authenticationScheme)
             BackchannelTimeout = TimeSpan.FromMinutes(1);
             Caption = OpenIdConnectAuthenticationDefaults.Caption;
             GetClaimsFromUserInfoEndpoint = false;
-            ProtocolValidator = new OpenIdConnectProtocolValidator();
+            ProtocolValidator = new OpenIdConnectProtocolValidator() { RequireState = false };
```

Comment:
I thought that the 'state' issue would block us, hence the temporary flag. Once you have the OIDC sample app working, then I vote ship it, we can deal with #404 later.